cmd/kyma/apply/apply.go

```diff
@@ -0,0 +1,19 @@
+package apply
+
+import (
+	"github.com/kyma-project/cli/cmd/kyma/apply/function"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new function command
+func NewCmd(o *cli.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "apply",
+		Short: "Applies local resources to the Kyma cluster.",
+		Long:  "Use this command to apply the local files and folders from your project to the Kyma cluster.",
```

Comment:
This description needs to be more generic. In the future we will have other apply commands that will not necessarily apply files and folders from a project.

Reply:
ok, good point